src/nuke/GradeBeauty.cpp

```diff
@@ -34,6 +34,9 @@ static const char* DEFAULT_VALUE_PYSCRIPT =
     "        knob.setDefaultValue(defaultValue)\n"
     "        knob.fromScript(knobToScript)"
 ;
+static const char* autoLabelScript =
```

Comment:
Shouldn't this be somehow defined in Python ? It feels weird to have Python code in C++.

Reply:
totally agree, I can't find any equivalent _setDefaultValue_ method for knobs in the Nuke headers.